docs/dictionary/function/processor.lcdoc

```diff
@@ -23,17 +23,15 @@ if the processor is "ARM" then ...
 
 Returns (enum):
 
--   x86: Windows systems (any architecture) and Mac OS X and Linux
-    systems with a 32 bit processor.
--   x86_64: Mac and Linux systems running a 64 bit processor
--   i386: iOS Simulator.
--   ARM: iOS and Android Devices and the Android simulator
-
+-   x86: 32 or 64 bit Windows systems and 32 bit macOS or Linux systems.
```

Comment:
It's probably not backwards compatible but `the applicationArchitecture` would be nicer for what the `the processor` currently does. Either way the docs need to spell this out because I didn't know until I checked the source and my guess is most people wouldn't. Not that I've ever really had an occasion to use it so maybe that's the reason I hadn't figured it out...